dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -62,29 +61,24 @@
     private HostCache hostCache = CacheLocator.getHostCache();
     private Host systemHost;
     private final SystemEventsAPI systemEventsAPI;
-    private final String contentTypeCondition;
-    private StringBuilder queryBuffer;
-    private final User systemUser;
+    private final String CONTENT_TYPE_CONDITION = "+contentType";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)